bower.json

```diff
@@ -1,49 +0,0 @@
-{
```

Comment:
@david-martin @sub-mod I know you're both using origin-web-common libraries. Is it a problem if we drop bower support completely in favor of yarn?

Reply:
@spadgett We did a standalone proof of concept that used origin-web-common last year, however that was dropped in favour of UI changes in the console instead.
No issue with changing to yarn